discord/no-decay/index.html

```diff
@@ -185,66 +185,6 @@ <h2 class="title is-2">Our Teams</h2>
                                 </div>
                             </div>
                         </div>
-                        <!-- <div class="column">
```

Comment:
Would be better if this was put into a scraps file. I kind of want to keep this code in case we want to use it later.

Reply:
Again source control is there for this exact thing :) no need to muddy up the code base with random files